bin/crew

```diff
@@ -228,18 +228,33 @@ def generate_compatible
 end
 
 def search (pkgName, silent = false)
-  pkgPath = CREW_PACKAGES_PATH + pkgName + '.rb'
+
+  if pkgName.start_with? 'ruby_'
```

Comment:
​```ruby
case​ ​pkgName​
​when​ ​/ruby/​
...
​else​
...
​end
```

Reply:
Okay that you! I couldn't figure that out earlier